app/assets/javascripts/pageflow/editor/views/nested_files_view.js

```diff
@@ -0,0 +1,81 @@
+pageflow.NestedFilesView = Backbone.Marionette.ItemView.extend({
```

Comment:
This could also have a domino based test, since it interacts with editor api. We could test that the specified columns actually appear, like we do in the `UploadableFilesView`.